common/libs/keystore.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { randomBytes, createCipheriv } from 'crypto';
+import { privateToPublic, publicToAddress, sha3 } from 'ethereumjs-util';
+import scrypt from 'scryptsy';
+import uuid from 'uuid';
+import { makeBlob } from 'libs/globalFuncs';
+
+export const scryptSettings = {
+  n: 1024
+};
+
+export const kdf = 'scrypt';
```

Comment:
The string.

https://github.com/kvhnuke/etherwallet/blob/f864c72022e571f560bab1beb919113c66e3308d/app/scripts/globalFuncs.js#L147